src/Controller/Organization/User/UserListController.php

```diff
@@ -41,6 +47,7 @@ public function __invoke(Request $request, Organization $organization): Response
         return $this->render('organization/user/user-list.html.twig', [
             'organization' => $organization,
             'users' => $this->userRepository->findByOrganization($organization),
+            'importantSkills' => $this->skillSetDomain->getImportantSkills(),
```

Comment:
you should maybe create a twig filter instead of injecting data to the templates